eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -178,7 +178,6 @@ jobs:
               BuildTargetingString: ${{ parameters.BuildTargetingString }}
 
   - job: 'RunRegression'
-    condition: and(succeededOrFailed(), or(eq(variables['Run.Regression'], 'true'), and(eq(variables['Build.Reason'], 'Schedule'), eq(variables['System.TeamProject'],'internal'))))
```

Comment:
@scbedd @praveenkuttappan I assume we don't want to remove this condition.

Reply:
I'm removing it temporarily - for testing purposes (forcing regression to run). I will undo this before merging.